Code/Engine/RendererFoundation/Shader/ShaderByteCode.h

```diff
@@ -42,7 +42,7 @@ struct EZ_RENDERERFOUNDATION_DLL ezShaderConstant
 
   static ezUInt32 s_TypeSize[Type::ENUM_COUNT];
 
-  void CopyDataFormVariant(ezUInt8* pDest, ezVariant* pValue) const;
+  void CopyDataFormVariant(ezUInt8* pDest, const ezVariant* pValue) const;
```

Comment:
While your at it, can you fix the typo in CopyDataFromVariant ?

Reply:
Fixed!